nbgrader/docs/source/user_guide/installation.rst

```diff
@@ -14,70 +14,69 @@ Or, if you use `Anaconda <https://www.anaconda.com/download>`__::
     conda install jupyter
     conda install -c conda-forge nbgrader
 
-nbgrader extensions
--------------------
 
-**Take note:** If you install nbgrader via `Anaconda
-<https://www.anaconda.com/download>`__ the nbgrader extensions will be
-installed and enabled for you upon installation. See the `Installation
-options`_ and `Disabling extensions`_ sections below for more information on
-changing the default installation option ``--sys-prefix`` or disabling one or
-more extensions.
+nbgrader extensions in Jupyter Lab
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Installing and activating extensions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Extensions will be automatically installed after the installation of nbgrader.
 
-You can install the nbgrader extensions for Jupyter notebook. Previously
-this was done using the ``nbgrader extension install`` command. However, moving
-forward this is done using the ``jupyter nbextension`` and ``jupyter
-serverextension`` commands.
+The installation will activated 4 server extensions
+(formgrader, assignment_list, course_list and validate_assignment)
+and 5 labextensions (formgrader, assignment_list, course_list, validate_assignment and create_assignment).
 
-To install and enable all nbextensions (**assignment list**, **create
-assignment**, **formgrader**, and **validate**) run::
+The server extensions can be disabled individually by running::
 
-    jupyter nbextension install --sys-prefix --py nbgrader --overwrite
-    jupyter nbextension enable --sys-prefix --py nbgrader
-    jupyter serverextension enable --sys-prefix --py nbgrader
+    jupyter server extension disable nbgrader.server_extensions.formgrader
+    jupyter server extension disable nbgrader.server_extensions.assignment_list
+    jupyter server extension disable nbgrader.server_extensions.course_list
+    jupyter server extension disable nbgrader.server_extensions.validate_assignment
 
-To work properly, the **assignment list**, **formgrader**, and **validate**
-extensions require both the nbextension and serverextension. The **create
-assignment** extension only has an nbextension part.
+To disable all the labextensions run::
+
+    jupyter labextension disable nbgrader
```

Comment:
Perhaps remove this, as it doesn't seem to work as one would expect, and just tell people they have to enable/disable the extensions individually.

Reply:
Done in https://github.com/jupyter/nbgrader/pull/1588/commits/5519c65b14e49e0712f690357e2bc7de597f26bd